frontend/src/lib/components/LemonSelect.tsx

```diff
@@ -6,24 +6,23 @@ import { PopupProps } from './Popup/Popup'
 import './LemonSelect.scss'
 import clsx from 'clsx'
 
-export interface LemonSelectOption {
+export interface LemonSelectOption<T> {
+    key: T
```

Comment:
I know this is WIP but I definitely agreed in our conversation that we should rename this to `value` to be closer to the native select options and not conflict wit React "keys"

Reply:
Ah, yes, good reminder! 📈